VMBackup/main/parameterparser.py

```diff
@@ -102,3 +106,13 @@ def __init__(self, protected_settings, public_settings, backup_logger):
         except Exception as e:
             errorMsg = "Exception occurred while populating includeLunList, Exception: %s" % (str(e))
             backup_logger.log(errorMsg, True)
+        
+        if(self.settings != None):
+            for flag in self.settings.keys():
+                foundFlag = False
+                for wellKnownFlag in self.wellKnownSettingFlags:
+                    if wellKnownFlag.lower() == flag.lower():
+                        foundFlag = True
+                        break
+                if(foundFlag != True):
+                    backup_logger.log("The flags passed through settings dictionary " + str(flag) +" is different from the expected one", True)
```

Comment:
str(flag) +" is not an expected setting name." is a better message and consistent with Windows extension message.